db/schema.sql

```diff
@@ -2233,13 +2233,6 @@ END
 $$;
 
 
---
```

Comment:
Needs a rebase maybe? I don't think you meant to check in a schema change.

Reply:
I think this might be fine - got the same thing in #425 - looks like maybe we forgot to run `make test` before our last merge